parsel/selector.py

```diff
@@ -260,23 +265,25 @@ class Selector:
 
     def __init__(
         self,
-        text: Optional[str] = None,
+        text: Optional[Union[str, bytes]] = None,
```

Comment:
I know I'm a bit late, but would it be possible to use another argument name to pass bytes, instead of allowing text to be either str or bytes? This is one of the big issue which was being solved in Python 3 (vs 2), removal of mixed use of bytes vs str. 

My worry is that some users may pass bytes unintentionally to Selector, and it'd silently cause decoding issues (a wrong encoding might be used). By having another argument instead of `text` where user can pass bytes data (similar to how we allow passing `root`, we can ensure that it's a feature which is not used accidentally.